examples/network/nodeStyles/imagesWithBordersAndPadding.html

```diff
@@ -0,0 +1,105 @@
+<!doctype html>
```

Comment:
This example should be added to the example overview page!

Reply:
@mojoaxel added here https://github.com/r3code/vis-network/commit/eef823ee3b3d20860708563d3b810ec116abdc41